src/OrchardCore.Modules/Orchard.ContentPreview/Content/Scripts/contentpreview.edit.js

```diff
@@ -68,4 +68,9 @@ $(function () {
         // to pre-render the view even if no contentpreview:render is already sent
         sendFormData();
     });    
+
+    $(window).on('unload', function(){
+        // this will raise an event in the preview window to notify that the live preview is no longer active.
+        localStorage.setItem('contentpreview:not-connected:' + previewId, '');
```

Comment:
I've been reading and I think that the only way would be by opening the preview through javascript. Something like this: 
`
var childWindow = window.open(urlToPreview);
$(childWindow).trigger( some event in the child);
`
And from the child-preview we could access the parent-admin through the 'window.opener" property that is automatically populated.

I found this post useful: [Easy Inter-window Communications with jQuery](https://objectpartners.com/2012/07/31/easy-inter-window-communications-with-jquery/)
Right now we use an anchor tag to open the preview window.
Do you think it is a good idea to mix both approaches to cross-window communication ( jquery.trigger vs storage  event) ?
The storage event way seems to work fine. But if you think it is better to use the $(window).trigger approach I can investigate further.